pandas/_libs/tslibs/timestamps.pyx

```diff
@@ -195,6 +195,11 @@ class Timestamp(_Timestamp):
     nanosecond : int, optional, default 0
         .. versionadded:: 0.23.0
     tzinfo : datetime.tzinfo, optional, default None
+    fold : int, default is None
```

Comment:
add a versionadded 1.1 here (give the possibliites), e.g. {0, 1}

Reply:
Done.